### PR TITLE
add comments with datastore

### PR DESCRIPTION
**Pull request by sarahz916**
Completed step 5 of week-3 walkthrough

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -25,25 +31,43 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
- ArrayList<String> comments = new ArrayList<String>();
+;
     
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (kkincy) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:36]: Do you want to add a code comment for what this block of code generally is intended to do? It's generally a good coding practice.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -70,7 +70,7 @@ <h2>Comments</h2>
     
     <h2> Fetch comment information </h2>
       <p>Click here to get comment info:</p>
-     <ul id="Comments"></ul>
+     <ul id="Comments"></ul> <!-- make div and then make seperate boxes of p to look more like comments see BS documentation-->
```
Comment (kkincy) [inline portfolio/src/main/webapp/index.html]: Is this a TODO? Make sure to add TODO in the comment. The convention at Google for C++ is usually TODO(kkincy).

File: portfolio/pom.xml [hunk 3]
```diff
@@ -23,11 +23,18 @@
       <version>4.0.1</version>
```
Comment (kkincy) [inline portfolio/pom.xml:23]: A more general comment is to add more information in the description of this pull request. Basically, what you're changing, refactoring, or implementing, and any TODOs remaining.

**Review by kkincy — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -25,25 +31,43 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
- ArrayList<String> comments = new ArrayList<String>();
+;
```
Comment (JKubs15) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Did you mean to leave this floating semicolon?

**Pull request merged**